lib/generators/react_on_rails/setup_files_generator.rb

```diff
@@ -0,0 +1,26 @@
+require "rails/generators"
+
+module ReactOnRails
+  module Generators
+    class SetupFilesGenerator < Rails::Generators::Base
+      source_root File.expand_path("../templates", __FILE__)
+
+      def update_git_ignore
+        append_to_file ".gitignore" do
+          "# React on Rails\nnpm-debug.log\nnode_modules"
```

Comment:
@justin808 if I add the linters task, rake assets:precompile bombs out with 'LoadError: cannot load such file -- scss_lint/rake_task'. I've traced it to https://github.com/shakacode/react-webpack-rails-tutorial/blob/master/Gemfile#L73

Should we require that gem to be added or should I not add the linters task?